### DISCUSSION TITLE
Different schema in postgres but complaining of duplicate table names

### DISCUSSION THREAD

**Discussion by vincent-contreras** [mikro-orm/mikro-orm#4332] [Q&A] [answered]
I am asking about this commit: https://github.com/mikro-orm/mikro-orm/commit/069302936bfa1fd10aeb2636ccdc3ca6ea021c4b

I have code in which there are two entities with the same names but different schemas. Because of this, I cannot do `MikroOrm.init()`.

Can somebody please work on when checking for duplicate table names, it should also check if it's in the same schema?

**Comment by B4nan** [comment]
Oh good catch, that's a bug for sure, it should respect the schema name. 

You can disable this validation for now, via `discovery.checkDuplicateTableNames` option.

**Reply by B4nan** [reply, marked as answer]
This should help https://github.com/mikro-orm/mikro-orm/commit/5b89b0056b6f87e6d5f4960bb2395d37bb9b3c28

**Reply by vincent-contreras** [reply]
Thank you very much for the bug fix!
